app/code/Magento/Checkout/view/frontend/layout/default.xml

```diff
@@ -56,7 +56,14 @@
                         </item>
                     </argument>
                 </arguments>
-                <container name="minicart.addons" label="Mini-cart promotion block"/>
+                <container name="minicart.addons" label="Mini-cart promotion block">
+                    <block class="Magento\Checkout\Block\ProductPricesUpdated" name="checkout.cart.product.prices.updated"
```

Comment:
You should define ttl argument on the block in order to cache this block separately in Varnish and not flush entire full page cache.